lesson4/_homework/task_01/script.js

```diff
@@ -0,0 +1,22 @@
+
+//                    Adapt Academy 2019 lesson 04 - JS | task 01 by Donatas Dereškevičius                     
+
+const countryList = [ 
+    {"name": "Afghanistan", "code": "AF"}, 
+    {"name": "Åland Islands", "code": "AX"}, 
+    {"name": "Albania", "code": "AL"}, 
+    {"name": "Algeria", "code": "DZ"}
+    ]
+
+    const getCountryByCode = (data) => {
+
+        input = document.querySelector(".form-input").value;
+        data = input.toUpperCase();
+
+
+        for(let {code: x, name: y} of countryList) {
+            if(x === data){
```

Comment:
Try to avoid variables or object properties as x or y. They are very non descriptive.